src/Client/BasicClient/BasicClientImpl.php

```diff
@@ -104,14 +106,14 @@ protected function getSocketMessenger(): SocketMessenger
     /**
      * @param AuthKey       $authKey
      * @param Proxy|null    $proxy
-     * @param callable|null $cb      function()
+     * @param callable|null $cb           function()
+     * @param bool          $updateStatus
      *
      * @throws TGException
      *
      * @return void
-     * @return void
      */
-    public function login(AuthKey $authKey, ?Proxy $proxy = null, callable $cb = null)
+    public function login(AuthKey $authKey, ?Proxy $proxy = null, callable $cb = null, bool $updateStatus = true)
```

Comment:
Better to move this functionallity out of basic client into some extended client

Reply:
Fixed, please take a look.